docs/manuals/source/TasksAndConcepts/UsingTapeDrivesWithoutAutochanger.rst

```diff
@@ -3,7 +3,7 @@
 Using Tape Drives without Autochanger
 =====================================
 
-.. index
+.. index::
    single: Strategy; Backup
    single: Backup Strategies
```

Comment:
```suggestion
   single: Backup; Strategies
```

Do you think there should be a semicolon ? We have lots of indices under `Backup` already so this would fit.

Reply:
See for example the next index invocation directly beneath this:
`single: Backup; One Tape`